### PR TITLE
Add support for Django 1.10

### PR DESCRIPTION
**Pull request by yakky**

### CODE REVIEW

**Comment by yakky** [comment]
@czpython could you review this?

**Comment by yakky** [comment]
@czpython @stefanfoulis I'd like a review from one of you

File: aldryn_apphooks_config/mixins.py [hunk 1]
```diff
@@ -15,8 +15,10 @@ class AppConfigMixin(object):
     """
     def dispatch(self, request, *args, **kwargs):
         self.namespace, self.config = get_app_instance(request)
+        request.current_app = self.namespace
         return super(AppConfigMixin, self).dispatch(request, *args, **kwargs)
 
     def render_to_response(self, context, **response_kwargs):
-        response_kwargs['current_app'] = self.namespace
+        if 'current_app' in response_kwargs:  # pragma: no cover
+            response_kwargs['current_app'] = self.namespace
```
Comment (czpython) [inline aldryn_apphooks_config/mixins.py]: what would be the value of `current_app` in `response_kwargs` if not `self.namespace`?

Comment (yakky) [inline aldryn_apphooks_config/mixins.py]: Then the project setup is really broken as ``current_app`` and the request namespace does not match (see https://github.com/aldryn/aldryn-apphooks-config/blob/master/aldryn_apphooks_config/utils.py#L26)
And anyway we need current_app to be the same as namespace :)

**Pull request merged**